api/v1_projects.inc

```diff
@@ -294,6 +306,7 @@ function render_project_json($project, $return_fields = null)
             "last_state_change_time" => date(DATE_ATOM, $project->modifieddate),
             "last_page_done_time" => date(DATE_ATOM, $project->t_last_page_done),
             "last_edit_time" => date(DATE_ATOM, $project->t_last_edit),
+            "smoothread_deadline" => date(DATE_ATOM, $project->smoothread_deadline),
```

Comment:
We need to add this to the `#/components/schemas/project` schema as well.

Reply:
TIL. Will do!